tests/github/data.json

```
@@ -21,5 +21,496 @@
+    }
+  ],
+  "commit_comment": [
+    {
+      "event_type": "commit_comment",
+      "raw_json": {
+        "action": "created",
+        "comment": {
+          "html_url": "https://github.com/example-org/example-repo/commit/4d93b5294b201237#commitcomment-12345678",
+          "user": {
+            "login": "example-user"
+          },
+          "commit_id": "4d93b5294b201237",
+          "body": "comment content"
+        },
+        "repository": {
+          "full_name": "example-org/example-repo",
+          "html_url": "https://github.com/example-org/example-repo"
+        },
+        "sender": {
+          "login": "example-user"
+        }
+      }
+    },
+    {
+      "comments": ["comment content"],
+      "commits": ["<|https://github.com/example-org/example-repo/commit/4d93b529>"],
+      "links": ["<https://github.com/example-org/example-repo/commit/4d93b5294b201237#commitcomment-12345678|None>"],
+      "repo": "<https://github.com/example-org/example-repo|example-org/example-repo>",
+      "type": "EventType.COMMIT_COMMENT",
+      "user": "<https://github.com/example-user|example-user>"
+    }
+  ],
+  "fork": [
+    {
+      "event_type": "fork",
+      "raw_json": {
+        "forkee": {
+          "owner": {
+            "login": "user-who-forked"
+          },
+          "html_url": "https://github.com/user-who-forked/example-repo"
+        },
+        "repository": {
+          "full_name": "example-org/example-repo",
+          "html_url": "https://github.com/example-org/example-repo"
+        }
+      }
+    },
+    {
+      "links": ["<https://github.com/user-who-forked/example-repo|None>"],
+      "repo": "<https://github.com/example-org/example-repo|example-org/example-repo>",
+      "type": "EventType.FORK",
+      "user": "<https://github.com/user-who-forked|user-who-forked>"
+    }
+  ],
+  "push": [
+    {
+      "event_type": "push",
+      "raw_json": {
+        "ref": "refs/heads/branch-name",
+        "repository": {
+          "full_name": "example-org/example-repo",
+          "html_url": "https://github.com/example-org/example-repo"
+        },
+        "pusher": {
+          "name": "user-who-pushed"
+        },
+        "sender": {
+          "login": "user-who-pushed"
+        },
+        "commits": [
+          {
+            "id": "f30421319e41a3a",
+            "message": "commit-message1"
+          },
+          {
+            "id": "5g0521417e40i37d9",
+            "message": "commit-message2"
+          }
+
+        ]
+      }
+    },
+    {
+      "commits": ["<commit-message1|https://github.com/example-org/example-repo/commit/f30421319e41a3a>", "<commit-message2|https://github.com/example-org/example-repo/commit/5g0521417e40i37d9>"],
+      "ref": "branch-name",
+      "repo": "<https://github.com/example-org/example-repo|example-org/example-repo>",
+      "type": "EventType.PUSH",
+      "user": "<https://github.com/user-who-pushed|user-who-pushed>"
+    }
+  ],
+  "star-add": [
```

Name all test cases consistently. use underscores.
```suggestion
  "star_add": [
```
Do the same for others too.